docs/core/tools/dotnet-install-script.md

```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
 
 The installation scripts download the ZIP/tarball file from the CLI build drops and proceed to install it in either the default location or in a location specified by `-InstallDir|--install-dir`. By default, the installation scripts download the SDK and install it. If you wish to only obtain the shared runtime, specify the `--shared-runtime` argument.
 
 By default, the script adds the install location to the $PATH for the current session. Override this default behavior by specifying the `--no-path` argument.
 
 Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-You can install a specific version using the `--version` argument. The version must be specified as a 3-part version (for example, 1.0.0-13232). If omitted, it uses the `latest` version.
+You can install a specific version using the `--version` argument. The version must be specified as a three-part version (for example, 1.0.0-13232). If not provided, it uses the `latest` version.
 
 ## Options
 
 `-Channel <CHANNEL>`
 
 Specifies the source channel for the installation. The possible values are:
 
-- `Current` - Current release
-- `LTS` - Long-Term Support channel (current supported release)
-- Two-part version in X.Y format representing a specific release (for example, `2.0` or `1.0`)
-- Branch name [for example, `release/2.0.0`, `release/2.0.0-preview2`, or `master` for the latest from the `master` branch ("bleeding edge" nightly releases)]
+* `Current` - Most current release.
+* `LTS` - Long-Term Support channel (most current supported release).
+* Two-part version in X.Y format representing a specific release (for example, `2.0` or `1.0`).
+* Branch name. For example, `release/2.0.0`, `release/2.0.0-preview2`, or `master` (for nightly releases).
 
-The default value is `LTS`. For more information on .NET support channels, see the [.NET Core Support Lifecycle](https://www.microsoft.com/net/core/support) topic.
+The default value is `LTS`. For more information on .NET support channels, see the [.NET Support Policy](https://www.microsoft.com/net/platform/support-policy#dotnet-core) page.
 
 `-Version <VERSION>`
 
 Represents a specific build version. The possible values are:
 
-- `latest` - Latest build on the channel (used with the `-Channel` option)
-- `coherent` - Latest coherent build on the channel; uses the latest stable package combination (used with Branch name `-Channel` options)
-- Three-part version in X.Y.Z format representing a specific build version; supersedes the `-Channel` option. For example: `2.0.0-preview2-006120`
+* `latest` - Latest build on the channel (used with the `-Channel` option).
```

Comment:
I think that I'd consider indenting everything belonging to a version. Currently, the options don't really stand out from their related text.

Reply:
I'm experimenting using bold for the options like the Azure CLI docs style. I'll see if that helps. If it does, I'll apply the same change to the other CLI commands on a separate PR.